platform/linux-generic/m4/odp_pcapng.m4

```diff
@@ -0,0 +1,19 @@
+##########################################################################
+# Enable PCAPNG support
+##########################################################################
+have_pcapng=no
+AC_ARG_ENABLE([pcapng-support],
+	[AS_HELP_STRING([--enable-pcapng-support],
+	[enable experimental tcpdump for pktios])],
+	have_pcapng=$enableval)
+
+if test x$have_pcapng = xyes
+then
+    AC_DEFINE([ODP_PCAPNG], [1],
+	      [Define to 1 to enable pcapng support])
+else
+    AC_DEFINE([ODP_PCAPNG], [0],
+	      [Define to 0 to disable pcapng support])
+fi
```

Comment:
Just use one AC_DEFINE and check using `#ifdef` inside.

Reply:
We did that in net_mdev and the if was corrected to check the value was equal to 1

Reply:
Anyway, just call AC_DEFINE once.